testnets/jackaltestnet2/assetlist.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "jackaltestnet",
+  "assets": [
+    {
+      "description": "The native staking and governance token of Jackal.",
+      "denom_units": [
+        {
+          "denom": "ujkl",
+          "exponent": 0
+        },
+        {
+          "denom": "jkl",
+          "exponent": 6
+        }
+      ],
+      "base": "ujkl",
+      "name": "Jackal",
+      "display": "jkl",
+      "symbol": "JKL",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/jackaltestnet2/images/jkl.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/jackaltestnet2/images/jkl.svg"
+      },
+      "coingecko_id": "jackal",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/jackaltestnet2/images/jkl.png",
```

Comment:
Please remove the duplicate uploads, and use image sync instead (they already exist at ~/jackal/images/).